### PR TITLE
Add a blog post about creme de la pods

### PR DESCRIPTION
**Pull request by orta**
Added a blog post announcing @CremeDeLaPods now that it's been a week - @CocoaPods/core

### CODE REVIEW

File: _posts/2015-11-07-Creme-de-la-Pods.markdown [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title:  "Introducing: Creme de la Pods"
+author: orta
+categories: twitter cocoapodsfeed cocoapods
+---
+
+TL;DR: _You could always find out what was new with [@CocoaPodsFeed](https://twitter.com/cocoapodsfeed) and [feeds.cocoapods.org](https://feeds.cocoapods.org). Now you can see new CocoaPods that have a [QI](https://guides.cocoapods.org/making/quality-indexes.html) score of over 75 on [@CremeDeLaPods](https://twitter.com/cremedelapods)._
+
+Keeping on top of the community's work is a challenge, with ~13,500 CocoaPods available and roughly 20 new pods every day. We work on CocoaPods to encourage  [contributions to OSS](https://github.com/CocoaPods/CocoaPods#project-goals). A by-product of this is that if you follow the whole stream of new CocoaPods, it can get hard to distinguish between "my first library" and "paradigm changing."
```
Comment (k0nserv) [inline _posts/2015-11-07-Creme-de-la-Pods.markdown]: "paradigm swift" sounds more natural imo

Comment (orta) [inline _posts/2015-11-07-Creme-de-la-Pods.markdown]: haha, can't tell if joking - but might just work

File: _posts/2015-11-07-Creme-de-la-Pods.markdown [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+I have taken a stab at solving this problem, the first time we run documentation parsing for a Pod [CocoaDocs] (https://github.com/CocoaPods/cocoadocs-api/commit/3dd3ee32f253d485576fb49fa272945cd5d5462b) will send out a tweet as [@CremeDeLaPods](https://twitter.com/cremedelapods) with the exact same formatting as [@CocoaPodsFeed](https://twitter.com/CocoaPodsFeed) does.
```
Comment (AliSoftware) [inline _posts/2015-11-07-Creme-de-la-Pods.markdown]: … parsing for a Pod, if that Pod's QI is over 75 then CocoaDocs will…

Comment (orta) [inline _posts/2015-11-07-Creme-de-la-Pods.markdown]: very good point

File: _posts/2015-11-07-Creme-de-la-Pods.markdown (hunk 1 above)
Comment (k0nserv) [inline _posts/2015-11-07-Creme-de-la-Pods.markdown]: lol, that was no intentional, but yeah might work. I meant "paradigm shift" for the record

**Comment by k0nserv** [comment]
The only other points of feedback I have is that it ends a bit abruptly, maybe a conclusion of a paragraph or two would be a nice addition. The text "some examples" would be slightly better if bolded or made into a header

**Comment by orta** [comment]
Yeah, that's legit, I could improve this

**Comment by AliSoftware** [comment]
Where's the mandatory GIF ending that article?

File: _posts/2015-11-07-Creme-de-la-Pods.markdown [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title:  "Introducing: Creme de la Pods"
+author: orta
+categories: twitter cocoapodsfeed cocoapods
+---
+
+TL;DR: _You could always find out what was new with [@CocoaPodsFeed](https://twitter.com/cocoapodsfeed) and [feeds.cocoapods.org](https://feeds.cocoapods.org). Now you can see new CocoaPods that have a [QI](https://guides.cocoapods.org/making/quality-indexes.html) score of over 75 on [@CremeDeLaPods](https://twitter.com/cremedelapods)._
```
Comment (AliSoftware) [inline _posts/2015-11-07-Creme-de-la-Pods.markdown]: According to the source code the limit seems to actually be 70, not 75?

**Comment by orta** [comment]
Thanks gents, I've given it another shot :)

File: _posts/2015-11-07-Creme-de-la-Pods.markdown [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+So there should be no surprises here if you've seen tweets from [@CocoaPodsFeed](https://twitter.com/CocoaPodsFeed)
+
+<center>
+  <blockquote class="twitter-tweet" lang="en" data-cards="hidden"><p lang="en" dir="ltr">[Venice] CSP for Swift 2 (Linux ready) <a href="https://t.co/DdPyqojBjI">https://t.co/DdPyqojBjI</a></p>&mdash; Creme De La Pods (@CremeDeLaPods) <a href="https://twitter.com/CremeDeLaPods/status/663133105809698821">November 7, 2015</a></blockquote>
+
+  <blockquote class="twitter-tweet" lang="en" data-cards="hidden"><p lang="fr" dir="ltr">[ControllerContainer by <a href="https://twitter.com/3lvis">@3lvis</a>] View Controller Containment for humans <a href="https://t.co/3WgHkIrUUa">https://t.co/3WgHkIrUUa</a></p>&mdash; Creme De La Pods (@CremeDeLaPods) <a href="https://twitter.com/CremeDeLaPods/status/662624562778144770">November 6, 2015</a></blockquote>
+  </center>
+
+Every Pod also comes with an overview images too, making it easy to see some of the salient properties, we've been doing this for a few years.
```
Comment (AliSoftware) [inline _posts/2015-11-07-Creme-de-la-Pods.markdown]: With ~~an~~ some overview images
(Or: "with an overview image~~s~~"?)

File: _posts/2015-11-07-Creme-de-la-Pods.markdown [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+  <center>
+  <blockquote class="twitter-tweet" lang="en"><p lang="en" dir="ltr">[Money by <a href="https://twitter.com/danthorpe">@danthorpe</a>] Swift types for working with Money. <a href="https://t.co/jCdJlrZV9Z">https://t.co/jCdJlrZV9Z</a></p>&mdash; Creme De La Pods (@CremeDeLaPods) <a href="https://twitter.com/CremeDeLaPods/status/662573883728490496">November 6, 2015</a></blockquote> <script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+  <a class="twitter-follow-button" href="https://twitter.com/CremeDeLaPods" data-size="large">Follow @CremeDeLaPods</a>
+</center>
```
Comment (AliSoftware) [inline _posts/2015-11-07-Creme-de-la-Pods.markdown]: This link is already present a few lines above in the post, no need to display it twice.

File: _posts/2015-11-07-Creme-de-la-Pods.markdown [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+Keeping on top of the community's work is a challenge, with ~13,500 CocoaPods available and roughly 20 new pods every day. We work on CocoaPods to encourage  [contributions to OSS](https://github.com/CocoaPods/CocoaPods#project-goals). A by-product of this is that if you follow the whole stream of new CocoaPods, it can get hard to distinguish between "my first library" and "an awesome paradigm _swift_."
+
+I have taken a stab at solving this problem, by trying to select the best pods by their [Quality Index](https://guides.cocoapods.org/making/quality-indexes.html). The first time we run documentation parsing for a Pod, if that Pod's QI is over 70 then [CocoaDocs] (https://github.com/CocoaPods/cocoadocs-api/commit/3dd3ee32f253d485576fb49fa272945cd5d5462b) will send out a tweet on the [@CremeDeLaPods](https://twitter.com/cremedelapods) twitter feed, with the exact same formatting as [@CocoaPodsFeed](https://twitter.com/CocoaPodsFeed).
+
+<a class="twitter-follow-button" href="https://twitter.com/CremeDeLaPods">Follow @CremeDeLaPods</a>
+
+<!-- more -->
+
+So there should be no surprises here if you've seen tweets from [@CocoaPodsFeed](https://twitter.com/CocoaPodsFeed)
```
Comment (AliSoftware) [inline _posts/2015-11-07-Creme-de-la-Pods.markdown]: "So if you've seen tweets from CocoaPodsFeed, there should be no surprises on the look of those tweets:"

File: _posts/2015-11-07-Creme-de-la-Pods.markdown (hunk 4 above)
Comment (AliSoftware) [inline _posts/2015-11-07-Creme-de-la-Pods.markdown]: … to see some of the salient properties; we've been doing this…" (prefer semicolon here — Yeah that's nitpicking, but I'm the Guardian of Consistency after all 😄)

File: _posts/2015-11-07-Creme-de-la-Pods.markdown [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+layout: post
+title:  "Introducing: Creme de la Pods"
+author: orta
+categories: twitter cocoapodsfeed cocoapods
+---
+
+TL;DR: _You could always find out what was new with [@CocoaPodsFeed](https://twitter.com/cocoapodsfeed) and [feeds.cocoapods.org](https://feeds.cocoapods.org). Now you can see new CocoaPods that have a [QI](https://guides.cocoapods.org/making/quality-indexes.html) score of over 70 on [@CremeDeLaPods](https://twitter.com/cremedelapods)._
+
+Keeping on top of the community's work is a challenge, with ~13,500 CocoaPods available and roughly 20 new pods every day. We work on CocoaPods to encourage  [contributions to OSS](https://github.com/CocoaPods/CocoaPods#project-goals). A by-product of this is that if you follow the whole stream of new CocoaPods, it can get hard to distinguish between "my first library" and "an awesome paradigm _swift_."
+
```
Comment (AliSoftware) [inline _posts/2015-11-07-Creme-de-la-Pods.markdown]: Badum tsss! :trollface:

File: _posts/2015-11-07-Creme-de-la-Pods.markdown (hunk 5 above)
Comment (orta) [inline _posts/2015-11-07-Creme-de-la-Pods.markdown]: this one is bigger, and after you know what you're getting in for, it's staying ;)

**Comment by orta** [comment]
Alright, I'm shipping it and announcing in a few hours.

File: _posts/2015-11-07-Creme-de-la-Pods.markdown (hunk 5 above)
Comment (AliSoftware) [inline _posts/2015-11-07-Creme-de-la-Pods.markdown]: Ah, I only saw the rendering wishing GitHub, missed that `data-size="large"`! :+1:

**Comment by AliSoftware** [comment]
:ok_hand:

**Pull request merged**